clients/graphql-kotlin-client-serialization/src/main/kotlin/com/expediagroup/graphql/client/serialization/types/KotlinXGraphQLError.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.client.serialization.types
+
+import com.expediagroup.graphql.client.serialization.serializers.AnyKSerializer
+import com.expediagroup.graphql.client.serialization.serializers.GraphQLErrorPathSerializer
+import com.expediagroup.graphql.client.types.GraphQLError
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class KotlinXGraphQLError(
```

Comment:
Should we call this different errors or should we have the specific type just be in the package name? Should we keep as `KotlinGraphQLError`?

Reply:
Named those implementations differently to avoid the name clash when importing both interface and implementation. That being said if you prefer same names we can definitely rename those - probably would drop the "client" part in the interface names as well.